### ISSUE TITLE
Proofreading of {memo X} lines

### ISSUE THREAD

**Issue by TOSBase** [Treeofsavior/EnglishTranslation#1105]
When I checked the korean and english files ealier I notice something about the {memo X} lines.

ETC_20150714_012232 {memo X}팀(2vs2)
ETC_20150714_012233 팀(3vs3)
ETC_20150714_012234 {memo X}팀(4vs4)
ETC_20150714_012235 팀(5vs5)

ETC_20150714_012232 {memo X}Team (2vs2)
ETC_20150714_012233 {memo X}Team (3vs3)
ETC_20150714_012234 {memo X}Team (4vs4)
ETC_20150714_012235 Team (5vs5)

Not only is there an inconsitency, it also seems to me that the {memo X} got removed from the korean line ETC_20150714_012233. This might be because at first only 5vs5 was available and now 3vs3 got added. However this means that the {memo X} works different from what I (and maybe others) thought. I thought that only outdated lines would be marked with and X and these lines could be ignored and there would be no need to translate them. However if this X just marks all currently unused lines as it seems to me now, this also means that it will mark lines that will be added in the future like additional PVP modes and that lines with an X could be used in the future. If this is the case we also need to proofread those lines otherwise we will run into problems.

**Comment by Soukyuu** [comment]
If you look at the tag, it's `filename_date_linecounter`. Meaning, should the line be used again, we will get a new tag for it, say, `ETC_20151004_000001 Team (3vs3)`. That line will have to be translated/proofread again anyway, so there is no real reason to proofread memo Xd lines.

**Comment by TOSBase** [comment]
You're completely missing the point...

Look again at what I wrote because clearly you didn't the first time....

ETC_20150714_012233 팀(3vs3)
ETC_20150714_012233 {memo X}Team (3vs3)

It's not that complicated to notice the problem. In the english file it has a memo X and in the korean file it doesn't. I don't know for sure why but it's obviously a problem. The korean file is probably more reliable here so I'd guess that the english one is incorrect. The korean files are probably more or less autogenerated and lines which aren't used by the client are marked with the memo X.

A) This probably also includes lines which aren't used YET like the 3vs3 line. At first only 5vs5 was possible but all other texts were already added. So it seems like 3vs3 was enabled later so the memo X got removed from the korean files. If this is the case then all memo X lines might be used in the future and we need to proofread them.

B) Someone made a mistake and accidently added the memo X line in the english file. Then we still need to be careful because this means that other memo X lines could also be wrong (I noticed many differences between the korean and english files)

"New" texts get added as new line at the bottom of the file (changed like count as new here) but this obvously doesn't apply to a line was isn't changed at all.
